### PR TITLE
Rwc2017

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: src/action_server/actions/find.py [hunk 1]
```diff
@@ -21,8 +23,7 @@ class Find(Action):
     def __init__(self):
         Action.__init__(self)
         # TODO: change this to a python dictionary schema
-        self._required_field_prompts = {'object': " What exactly would you like me to find? ",
-                                        'location': " Where should I look? "}
+        self._required_field_prompts = {'object': " What exactly would you like me to find? "}
         self._required_skills = ['head', 'base', 'rightArm', 'speech']
 
     def _point_at_person(self, person):
```
Comment (LoyVanBeek) [inline src/action_server/actions/find.py]: I think you'll do yourself a favour when splitting this function up a bit

File: src/action_server/actions/find_out_and_report.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+from action import Action, ConfigurationData
+
+import rospy
+
+class FindOutAndReport(Action):
+    ''' The Say class implements the action to say something.
```
Comment (LoyVanBeek) [inline src/action_server/actions/find_out_and_report.py]: Copy-paste error

File: src/action_server/actions/say.py [hunk 3]
```diff
@@ -52,12 +91,22 @@ def _start(self):
...
                 "It's color is yellow and when you push the button, it turns red?         A chick in the blender"
             ])
+            self._execute_result.message = " I told a joke. "
+        elif self._sentence == 'SOMETHING_ABOUT_SELF':
+            line = random.choice([
+                "I once dragged a person across the floor for meters.",
+                "I am still a bit insecure about my gender with my manly voice and my white dress.",
+                "I've been to tournaments in seven countries already, and still counting. However, I feel like I'm getting a little too old for this shit.",
+                "I once tripped over the border of a Middle Size League soccer field at full speed."
+            ])
```
Comment (LoyVanBeek) [inline src/action_server/actions/say.py]: LOL

File: src/action_server/actions/find.py (hunk 1 above)
Comment (rokusottervanger) [reply src/action_server/actions/find.py]: The functionality is now working. I will refactor the individual actions in separate dedicated branches.

**Pull request merged**